Buildings/Fluid/BuriedPipes/package.order

```diff
@@ -0,0 +1,3 @@
+GroundCoupling
+Examples
+BaseClasses
```

Comment:
I don't think `Buildings.Fluid.FixedResistances.Validation.PlugFlowPipes.PlugFlowAIT` takes into account the capacity of the ground as `thickness=0.0032 "Pipe wall thickness"` and `CPip=length*((dh + 2*thickness)^2 - dh^2)*Modelica.Constants.pi/4*cPip*rhoPip`. Only the thermal resistance seems to be corrected to account for the ground. In that respect the `GroundCoupling` model brings higher fidelity as if we were to simulate uninsulated burried pipes with the former approach, the outlet temperature would exhibit variations with the same dynamics as the air temperature.

Reply:
Yes, I noticed that as well. I think this is a mistake, either in how the validation model instantiate the `PlugFlowPipe` objects, or in how the `PlugFlowPipe` class is defined.
The model distinguishes between the pipe wall and the insulation, and the insulation heat capacity is ignored in the calculation of the equivalent volume. This makes sense in most application since the heat capacity of the insulation will likely be insignificant compared to the pipe wall, but this simplification is likely erroneous in cases where the surrounding soil is encompassed in the insulation.

Reply:
@AntoineGautier Are you ok resolving this?

Reply:
Yes